packages/mds-transactions-service/@types/index.ts

```diff
@@ -91,6 +99,9 @@ export interface TransactionSearchParams {
   provider_id?: UUID
   start_timestamp?: Timestamp
   end_timestamp?: Timestamp
+  before?: string
+  after?: string
+  limit?: number
```

Comment:
@karcass 
Is it possible to add `vehicle_id` to the TripReceiptDetailsDomainModel?
Right now vehicle_id is only accessible by
`TransactionDomainModel -> TripRecepitDetailsDomainModel -> VehicleEvent -> Device`, but `Device` has to be fetched separately (VehicleEvent only references Device by id). This is possible from the front end, but will require additional requests for a single field.